### PR TITLE
bugfix - ignore duplicate fields

### PR DESCRIPTION
**Pull request by joefitter**
If a field appears on multiple steps in the same section, only the first one should be used. This is a bug which occurs when a journey has a potential alternate step containaing one or more fields that could be in a different part of the journey

### CODE REVIEW

File: test/fixtures/steps.js [hunk 1]
```diff
@@ -51,6 +51,14 @@ module.exports = {
       section: 'enquiry-details'
     }
   },
+  '/additional-2': {
+    fields: [
+      'additions-names'
```
Comment (easternbloc) [inline test/fixtures/steps.js]: I don't understand the test case for this? Is additions names used elsewhere?

This seems like a large breaking change...?

**Review by easternbloc — CHANGES_REQUESTED** [review]

File: test/fixtures/steps.js (hunk 1 above)
Comment (joefitter) [reply test/fixtures/steps.js]: additional names - typo derp

**Comment by joefitter** [comment]
@easternbloc it isn't a breaking change as it is an unwanted "feature". If things break elsewhere then they were affected by this bug. This basically avoids one field appearing multiple times in the summary under the same section (with the same value)

**Comment by easternbloc** [comment]
@joefitter can you add a test to cover this then that specifically explains the change?

**Comment by joefitter** [comment]
@easternbloc I didn't add anything specific as the fixture tests cover the bug by adding a duplicate field. I have added one to the stubbed helpers section checking no duplicates appear 🏩

**Comment by easternbloc** [comment]
@joefitter thanks this test helps for clarity

**Review by easternbloc — APPROVED** [review]

**Comment by joefitter** [comment]
💋
